freebasics/templates/core/section_page.html

```diff
@@ -15,7 +15,7 @@ <h6 class="block-heading fb-accent-1">{{self.title}}</h6>
   {% load_child_articles_for_section self as articles %}
   {% for article in articles %}
     <li>
-      <h6 class="fb-accent-2">{{article.title}} </h6>
+      <a href="{% pageurl article %}" class="fb-accent-2 link-button"><h6 class="fb-accent-2">{{article.title}}</h6></a>
```

Comment:
The `fb-accent-2` class is applied twice here, to the parent and the `h6` child.